cvmfs/server/cvmfs_server_resign.sh

```diff
@@ -35,12 +39,13 @@ cvmfs_server_resign() {
   [ -n "$whitelist_path" ] || check_multiple_repository_existence "$names"
 
   # sanity checks
-  is_root || die "Only root can resign repositories"
+  [ $sign_published -eq 0 ] || [ $whitelist_path -eq 1 ] || die "Cannot use both -w and -p"
```

Comment:
Isn't this still wrong?  The `whitelist_path` parameter carries a string not an integer.